docs/ibis-for-dplyr-users.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Did you mean to give this a large bold style?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I don't follow, can you explain what you mean here?</p>

Reply:
<p>This shows up weird in the ReviewNB UI, but is fine in the rendered docs, it's good to go!</p>